mappings/net/minecraft/world/gen/stateprovider/PillarBlockStateProvider.mapping

```diff
@@ -1,2 +1,3 @@
 CLASS net/minecraft/class_4655 net/minecraft/world/gen/stateprovider/PillarBlockStateProvider
-	FIELD field_21313 block Lnet/minecraft/class_2248;
+	FIELD field_21313 state Lnet/minecraft/class_2248;
```

Comment:
```suggestion
	FIELD field_21313 block Lnet/minecraft/class_2248;
```

Reply:
Yeah, even though this is serialised as `state`, it goes through block<->state conversion when it does that.

Reply:
ah, hadn't really noticed that.